public/components/security/main.tsx

```diff
@@ -55,11 +60,33 @@ export const WzSecurity = compose(
 
   const [selectedTabId, setSelectedTabId] = useState(tab && tab[1] || 'users');
 
+
   const listenerLocationChanged = () => {
     const tab = window.location.href.match(securityTabRegExp);
     setSelectedTabId(tab && tab[1] || 'users')
   }
-  // This allows to redirect to a Security tab if you click a Security link in menu when you're already in a Security section 
+
+  const checkRunAsUser = async () => {
+    const currentApi = AppState.getCurrentAPI();
+    try {
+      const ApiCheck = await GenericRequest.request('POST',
+        '/api/check-api',
+        currentApi
+      );
+      return ApiCheck.data.allow_run_as;
+
+    } catch (error) {
+      ErrorHandler.handle(error, 'Error checking the current API');
+    }
+  }
+  const [allowRunAs, setAllowRunAs] = useState();
+  useEffect(() => {
+    checkRunAsUser()
+      .then(result => setAllowRunAs(result))
+      .catch(error => console.log('Hubo un error'))
```

Comment:
Remove the log in Spanish and manage the error otherwise or do nothing in the `.catch`.